package.json

```diff
@@ -14,6 +14,9 @@
     "prewatch": "npm run clean",
     "lint": "standard 'src/**/*.js' 'src/**/*.jsx'",
     "test": "node --harmony_proxies node_modules/.bin/jest --coverage",
+    "pretest:visual-regression": "(cd test/specs/ && docker-compose pull && docker-compose up --build -d && sleep 5)",
```

Comment:
I've just learnt this trick : it allows to "isolate" the `cd` command in a new bash and stay in the current directory after the command. By the way I totally don't know if npm scripts are working the same way or not.